__snapshots__/src/tree-node/actions.test.js.md

```diff
@@ -8,22 +8,20 @@ Generated by [AVA](https://ava.li).
 
 > Snapshot 1
 
-    [
-      <Action
-        actionData={
-          {
-            action: 'action-0',
-            id: undefined,
-          }
+    <Action
```

Comment:
This file should not have changed. I'm not sure what happened. Are you using React 15 by any chance?

Reply:
Hey! 
Wow, you're super active, kudos :) 
I myself have been bumping my head against this for the past 30mins. Travis fails due to this test, and yeah - I haven't made anything odd to change the file, other than `yarn test --update-snapshots`.
React is at version @**16.8.3**

Reply:
You're using a recent version which also is suspicious 😈 . I'm wondering if the new React is discarding array if it only has one element and treating it as a single dom node (I'm thinking for efficiency). 

To test the theory out, we could add one more item to the array in the [test](https://github.com/dowjones/react-dropdown-tree-select/blob/develop/src/tree-node/actions.test.js#L9) and see if it renders the array back in the snapshot.

Reply:
Hahaha :) It's never boring with JavaScript's dependencies! :) 
Indeed, I included one more dummy object in the actions array, rebuilt the snapshots and Travis does not complain any more :)

Reply:
The plot thickens. 🤔 

I wonder if it's React or ava. If it's React, then it means more issues for me down the line. 😕